src/session.js

```diff
@@ -0,0 +1,38 @@
+let crypto = require('crypto');
+
+let sessions = {};
+
+function hydrate(request, sessionId) {
+  let session = sessions[sessionId];
+  request.serial = session.serial;
```

Comment:
What do you think about copying everything from `session` over to `request` so we don't have to worry about the fields going forward etc